package.json

```diff
@@ -1,7 +1,9 @@
 {
   "name": "cambridge-voting-locations",
   "version": "0.0.1",
-  "dependencies": {},
+  "dependencies": {
+    "bootstrap": "^3.3.7"
```

Comment:
Because we have bootstrap-sass as a bower component, we can remove the npm module dependency.